sphinx_immaterial/apidoc/python/default.py

```diff
@@ -0,0 +1,27 @@
+import sphinx.application
+
+from . import annotation_style
+from . import object_ids
+from . import synopses
+from . import parameter_objects
+from . import style_default_values_as_code
+from . import subscript_methods
+from . import attribute_style
+from . import napoleon_admonition_classes
+from . import strip_property_prefix
+from . import section_titles
+from . import autodoc_property_type
+from . import type_annotation_transforms
+from . import strip_self_and_return_type_annotations
```

Comment:
I assume some of these unused imports invoke the one-off patches without an ext `setup()`

Reply:
Yes that's correct.